src/ChatBody.css

```diff
@@ -0,0 +1,6 @@
+.chatBody__container{
```

Comment:
since we will use display grid in parent div
we will use flex-direction: column here

Reply:
done